src/pages/Login.vue

```diff
@@ -395,16 +397,32 @@ export default {
       // whether the phone number entered by the user is valid
       return this.phoneInput.toString().match(/^([0]|\+91)?[6-9]\d{9}$/g) != null;
     },
+    startCountdown(seconds) {
+      // counts seconds before enabling resend OTP button
+      this.counter = seconds;
+      const interval = setInterval(() => {
+          this.counter--;
+          if(this.counter == 0)
+          {
+            this.counting= false;
+            clearInterval(interval);
+          }
+      }, 1000);
+    },
     requestOtp() {
       // requests OTP for the first time
       UserAPIService.requestOtp(this.formattedPhoneInput);
       this.requestedOtp = true;
+      this.counting = true;
+      this.startCountdown(60);
       this.invalidOtp = false;
     },
     resendOtp() {
       // resends OTP on user request
       UserAPIService.requestOtp(this.formattedPhoneInput);
       this.resentOtp = true;
+      this.counting = true;
+      this.startCountdown(60);

```

Comment:
based on feedback above:
```suggestion
      this.startResendOTPTimer();
```